cmd/check-gh-automation/main.go

```diff
@@ -205,6 +210,16 @@ func checkRepos(repos []string, bots []string, ignore sets.Set[string], client a
 				}
 			}
 		}
+		isCollaborator, err := client.IsCollaborator(org, repo, cherrypickRobot)
+		if err != nil {
+			return nil, fmt.Errorf("error checking collaborator status for openshift-cherrypick-robot on %s/%s: %w", org, repo, err)
+		}
+		if !isCollaborator {
+			repoLogger.Infof("openshift-cherrypick-robot is not a collaborator on %s/%s", org, repo)
+			failing.Insert(orgRepo)
+		} else {
+			repoLogger.Infof("Confirmed: openshift-cherrypick-robot is a collaborator on %s/%s", org, repo)
+		}
```

Comment:
This section needs to take place inside the
```
if plugin.Name == cherrypickPlugin {
```

Reply:
Also, keep in mind that the bot needs to either be a collaborator **or** an org member. It is not necessary for it to be both.

Reply:
Restructured the code + add or check